src/Features/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -13,6 +14,11 @@ internal static class CSharpCodeStyleOptions
     {
         internal const string FeatureName = "CSharpCodeStyle";
 
+        // TODO: get sign off on public api changes.
         public static readonly Option<bool> UseVarWhenDeclaringLocals = new Option<bool>(FeatureName, "UseVarWhenDeclaringLocals", defaultValue: true);
+
+        public static readonly Option<SimpleCodeStyleOption> UseVarForIntrinsicTypes = new Option<SimpleCodeStyleOption>(FeatureName, "UseImplicitTypingForIntrinsics", defaultValue: SimpleCodeStyleOption.Default);
+        public static readonly Option<SimpleCodeStyleOption> UseVarWhenTypeIsApparent = new Option<SimpleCodeStyleOption>(FeatureName, "UseImplicitTypingWhereApparent", defaultValue: SimpleCodeStyleOption.Default);
```

Comment:
these names don't match:

UseImplicitTypingWhereApparent
UseVarWhenTypeIsApparent 

Pick something and use it consistently everywhere.

Also consider using nameof here.